stdlib/contrib/jsternberg/aggregate/aggregate.flux

```diff
@@ -48,7 +74,7 @@ builtin null : A
 builtin none : A
 
 // define will define an aggregate function.
-define = (init, reduce, compute, fill=null) => (column, fill=fill) => ({
+define = (init, reduce, compute, fill=null) => (column="_value", fill=fill) => ({
```

Comment:
What does the fill parameter do here?

Reply:
It defines which value should fill that column if it is empty. It doesn't work yet with `window()`.